test/maxcdn_test.rb

```diff
@@ -138,9 +139,6 @@ def test_purge_file
 
   def test_purge_files
     assert_equal({"foo.txt"=>{"foo"=>"bar"}, "bar.txt"=>{"foo"=>"bar"}}, @max.purge(12345, [ "foo.txt", "bar.txt" ]))
-  end
```

Comment:
The second test was never actually being run, the newer version of webmock helpfully threw a warning messaging.